tls_cgo.go

```diff
@@ -0,0 +1,11 @@
+// +build cgo
+
+package main
+
+import "crypto"
+import "github.com/letsencrypt/pkcs11key"
+
+
+func newpkcs11(module, tokenLabel, pin string, pubkey crypto.PublicKey) (interface{}, error) {
```

Comment:
And then just access the flags directly in here instead of passing args. Makes it harder to unit test, but it's kinda hard to unit test something that depends on an external hardware module anyway.